recipes/urlquote/meta.yaml

```diff
@@ -0,0 +1,85 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "urlquote" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed.
+  # url: https://github.com/urlquote/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 4b3920f7e2653e05c92f3b1b11e8fe992e767884556d095ee82ee1a4c8213a46
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    # - {{ compiler('c') }}
+    - rust
+    - setuptools_scm
+    - milksnake
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
```

Comment:
```suggestion
    - six
    - milksnake
```

Reply:
Noob question: There seems to be a milksnake package, but it does not seem to be in the 'conda_forge' index. Do I need to qualify this somehow?

https://anaconda.org/kargf/milksnake

Reply:
No, we need it in the conda-forge index. Conda packages are binaries and we don't know how the linked one was built.

Reply:
`milksnake` is a generic Python package, so you could add it to this PR. Also you might want to use `conda skeleton pypi milksnake` as this will use all PyPI information to make a conda-package.